src/GitVersion.BuildAgents.Tests/Agents/GitHubActionsTests.cs

```diff
@@ -75,26 +76,30 @@ public void GetCurrentBranchShouldHandleBranches()
     public void GetCurrentBranchShouldHandleTags()
     {
         // Arrange
+        this.environment.SetEnvironmentVariable("GITHUB_REF_TYPE", "tag");
         this.environment.SetEnvironmentVariable("GITHUB_REF", "refs/tags/1.0.0");
 
         // Act
         var result = this.buildServer.GetCurrentBranch(false);
 
         // Assert
-        result.ShouldBe("refs/tags/1.0.0");
+        result.ShouldBeNull();
     }
 
     [Test]
     public void GetCurrentBranchShouldHandlePullRequests()
     {
         // Arrange
+        this.environment.SetEnvironmentVariable("GITHUB_EVENT_NAME", "pull_request");
+        this.environment.SetEnvironmentVariable("GITHUB_HEAD_REF", "some-branch");
```

Comment:
head ref already has the `refs/heads/` removed, this mirrors that behavior. Same for base ref as well.